lib/irc/ClientPool.js

```diff
@@ -251,8 +251,9 @@ ClientPool.prototype.totalReconnectsWaiting = function (serverDomain) {
 ClientPool.prototype.updateActiveConnectionMetrics = function(server, ageCounter) {
     const clients = Object.values(this._virtualClients[server].userIds);
     clients.forEach((bridgedClient) => {
-        if (bridgedClient.isDead()) {
-            return; // We don't want to include dead ones.
+        if (!bridgedClient || bridgedClient.isDead()) {
+            // We don't want to include dead ones, or ones that don't exist.
```

Comment:
```
var isNonexistentOrDead = !bridgedClient || bridgedClient.isDead()
if (isNonexistentOrDead) {
```

same amount of lines, expresses semantics in code instead of comments

Reply:
Good point, I'll include this in the next release since it's non-critical (and saves us having to apply it to two branches) but definitely reads better.